examples/playlist/schemas/v0-0.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Out of curiosity; Is this a manually-written file? Or a code-generated one? 🤔

Reply:
I tried a few flavors, these were generated from https://github.com/invopop/jsonschema and then I manually created the three versions.